angel_system/activity_hmm/core.py

```diff
@@ -743,11 +773,219 @@ def score_raw_detections(X, Z, plot_results=False):
 
     if plot_results:
         fig = plt.figure(num=None, figsize=(14, 10), dpi=80)
-        plt.rc('font', **{'size': 22})
+        plt.rc('font', **{'size': 28})
         plt.rc('axes', linewidth=4)
         plt.plot(recall, precision, linewidth=6, label='Raw Detections')
         plt.xlabel('Recall', fontsize=40)
         plt.ylabel('Precision', fontsize=40)
         fig.tight_layout()
 
+        plt.xlim([0, 1.01])
+        plt.ylim([0, 1.01])
+        fig.tight_layout()
+
+        plt.gca().yaxis.set_minor_locator(AutoMinorLocator(2))
+        plt.tick_params(
+                axis="y",
+                which="major",
+                grid_color='lightgrey')
+        plt.tick_params(
+                axis="y",
+                which="minor",
+                grid_linestyle='--',
+                grid_color='lightgrey')
+        plt.grid(axis='y', which='both')
+        plt.gca().xaxis.set_minor_locator(AutoMinorLocator(2))
+        plt.tick_params(
+                axis="x",
+                which="major",
+                grid_color='lightgrey')
+        plt.tick_params(
+                axis="x",
+                which="minor",
+                grid_linestyle='--',
+                grid_color='lightgrey')
+        plt.grid(axis='x', which='both')
+        plt.legend(fontsize=20, loc=0)
+
     return auc
+
+
+def load_and_discretize_data(activity_gt: str,
+                             extracted_activity_detections: str,
+                             time_window: float, uncertain_pad: float):
+    """
+
+    Parameters
+    ----------
+    activity_gt : str
+        Path to activity ground truth feather file.
+    extracted_activity_detections : str
+        Path to detection json.
+    time_window : float
+        Time (seconds) to discretize to.
+    uncertain_pad : float
+        Time uncertainty padding during change of class.
+
+    Raises
+    ------
+    Exception
+        DESCRIPTION.
+
+    Returns
+    -------
+    TYPE
+        DESCRIPTION.
+
+    """
+    gt_f = pd.read_feather(activity_gt)
+    # Keys: class, start_frame,  end_frame, exploded_ros_bag_path
+
+    gt = []
+    RE_FILENAME_TIME = re.compile(r"frame_\d+_(\d+)_\d+.\w+")
```

Comment:
This is no longer used, I only see it in commented-out code, which could also be removed.

Reply:
Addressed.